File: Kattis/detaileddifferences/detaileddifferences.cpp
#include <iostream>

int main() {
    int n;
    std::cin >> n;

    while (n > 0) {
        std::string s1, s2;
        std::cin >> s1 >> s2;
        int s1_length = s1.length();

        std::cout << s1 << "\n" << s2 << "\n";

        for (int i = 0; i < s1_length; i++) {
            if (s1[i] == s2[i]) {
                std::cout << ".";
            }
            else {
                std::cout << "*";
            }
        }
        std::cout << "\n";
        n--;
    }
    return 0;
}

File: Kattis/detaileddifferences/detaileddifferences.py
n = int(input())

while n > 0:
    s1 = input()
    s2 = input()
    s1_length = len(s1)

    print(s1)
    print(s2)

    for i in range(s1_length):
        if s1[i] == s2[i]:
            print(".", end="")
        else:
            print("*", end="")
    print()
    n -= 1